### PR TITLE
Project documentation (ioana_miruna.badica)

### PR DESCRIPTION
**Pull request by jojomojo22**
### Pull Request Overview

This pull request adds the documentation for the project Smart Lamp

### TODO or Help Wanted

Let me know if I should modify anything else

### Author

Signed-off-by: Bădică Ioana-Miruna <ioana.badica22@gmail.com>

### CODE REVIEW

File: docs/project/2024/ioana_miruna.badica/index.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+# Smart Lamp
+The prototype of a lamp with multiple functionalities.
+
+:::info 
+
+**Author**: Bădică Ioana-Miruna <br>
```
Comment (alexandruradovici) [inline docs/project/2024/ioana_miruna.badica/index.md:6]:
```suggestion
**Author**: Bădică Ioana-Miruna \
```

File: docs/project/2024/ioana_miruna.badica/index.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+The project consists of a LED that turns on the light automatically when it detects movement and only when it is dark. Besides, it should still have its regular "lamp" function, so it will have a button to turn it on "smart" mode. 
+
+## Motivation
+
+I have chosen this project because I wanted to implement something that could turn into a bigger project later such as using a real lamp instead of a LED, or implementing new functions like (if I were to use a RGB LED) to add a remote controller to switch the colors of the light from far away.
+
+## Architecture 
+
+Components <br>
```
Comment (alexandruradovici) [inline docs/project/2024/ioana_miruna.badica/index.md:21]: Please use the correct markdown syntax for lists.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
Please fill the pull request template.

File: docs/project/2024/ioana_miruna.badica/index.md (hunk 2 above)
Comment (jojomojo22) [reply docs/project/2024/ioana_miruna.badica/index.md]: I solved the <br> problem in all places, I hope it is okay now. Also filled the pull request template. Please let me know if I should make any other changes. Thank you!

File: docs/project/2024/ioana_miruna.badica/index.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
+# Smart Lamp
+The prototype of a lamp with multiple functionalities.
+
+:::info 
+
+**Author**: Bădică Ioana-Miruna \
+**GitHub Project Link**: [Smart Lamp](https://github.com/jojomojo22/upb-fils-ma.github.io/blob/project_documentation/docs/project/2024/ioana_miruna.badica/index.md)
```
Comment (alexandruradovici) [inline docs/project/2024/ioana_miruna.badica/index.md:7]:
```suggestion
**GitHub Project Link**:  fill in here the link to your project repository
```

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/ioana_miruna.badica/index.md (hunk 3 above)
Comment (jojomojo22) [reply docs/project/2024/ioana_miruna.badica/index.md]: I made the changes.

File: docs/project/2024/ioana_miruna.badica/index.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+| Button | | [2 x 0.4 RON](https://www.optimusdigital.ro/ro/butoane-i-comutatoare/1119-buton-6x6x6.html) |
+| Jumper wires | | [8 RON](https://www.optimusdigital.ro/ro/fire-fire-mufate/12-set-de-cabluri-pentru-breadboard.html) |
+| Breadboard | | [5 RON](https://www.optimusdigital.ro/ro/prototipare-breadboard-uri/44-breadboard-400-points.html) |
+| LED | | [0.4 RON](https://www.optimusdigital.ro/ro/optoelectronice-led-uri/696-led-rou-de-3-mm-cu-lentile-difuze.html?search_query=led&results=818) |
+
+
+
+## Software
+
+| Library | Description | Usage |
```
Comment (alexandruradovici) [inline docs/project/2024/ioana_miruna.badica/index.md:76]: No libraries?

**Review by alexandruradovici — APPROVED** [review]

**Review by TeodoraMiu — APPROVED** [review]

**Pull request merged**